core-model-test/tests/src/test/java/org/jboss/as/core/model/test/host/ServerConfigTestCase.java

```diff
@@ -54,7 +53,7 @@ public void testAddServerConfigBadSocketBindingGroup() throws Exception {
 
         KernelServices kernelServices = createKernelServices("host.xml");
 
-        PathAddress pa = PathAddress.pathAddress(PathElement.pathElement(HOST, MASTER), PathElement.pathElement(SERVER_CONFIG, "server-four"));
+        PathAddress pa = PathAddress.pathAddress(PathElement.pathElement(HOST, "master"), PathElement.pathElement(SERVER_CONFIG, "server-four"));
```

Comment:
It looks like you missed my [comment](https://github.com/wildfly/wildfly-core/pull/5108#pullrequestreview-1044858446);  this and the others on the same class should be replaced by "primary"